src/components/YourCompany/NameTranslations/AddNameTranslation.vue

```diff
@@ -21,15 +21,24 @@
       <v-row>
         <v-col>
           <div class="action-btns">
-            <v-btn large color="primary"
+            <v-btn v-if="!isAddingTranslation" large outlined color="error"
+                id="name-translation-remove"
+                 @click="emitRemoveName(nameIndex)"
+              >
+                <span>Remove</span>
+              </v-btn>
+            <v-btn large color="primary" class="ml-auto"
               id="name-translation-btn-ok"
               :disabled="!nameTranslationForm"
               @click="addTranslation()"
```

Comment:
Might as well fix the other btn's while your here :)

Reply:
Fixed zee button spacing (hopefully)